### PR TITLE
Feat: add get delete members to a bucket

### PR DESCRIPTION
**Pull request by sunbryely-influxdata**
Add member to a bucket
Get all members of a bucket
Remove a user from a bucket

### CODE REVIEW

File: src/common/paths/buckets_bucketID_members_userID.yml [hunk 1]
```diff
@@ -3,23 +3,43 @@ delete:
   tags:
     - Buckets
   summary: Remove a member from a bucket
+  description: |
+    Removes a member from a bucket.
+
+    Use this endpoint to remove a user's member privileges from a bucket. This
+    will remove user access to the bucket.
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_members_userID.yml:10]: ?
```suggestion
    Use this endpoint to remove a user's member privileges from a bucket.
    This removes the user's `read` and `write` permissions for the bucket.
```

File: src/common/paths/buckets_bucketID_owners.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ post:
           $ref: "../schemas/AddResourceMemberRequestBody.yml"
   responses:
     "201":
-      description: Bucket owner added
+      description: Bucket owner adde
```
Comment (jstirnaman) [inline src/common/paths/buckets_bucketID_owners.yml:49]:
```suggestion
      description: Success. The user is an owner of the bucket.
```

File: src/common/schemas/UserResponse.yml [hunk 3]
```diff
@@ -2,12 +2,15 @@
     id:
       readOnly: true
       type: string
-    oauthID:
-      type: string
```
Comment (jstirnaman) [inline src/common/schemas/UserResponse.yml:6]: `oauthID` isn't valid?

**Review by jstirnaman — APPROVED** [review]
Suggestion and questions

File: src/common/schemas/UserResponse.yml (hunk 3 above)
Comment (sunbryely-influxdata) [reply src/common/schemas/UserResponse.yml:6]: `oAuthID` isn't returned by the response anymore in either cloud or oss

**Pull request merged**